pkg/tests/BUILD

```diff
@@ -3,12 +3,68 @@
 licenses(["notice"])
```

Comment:
There's no copyright statement in this file, but since this is preexisting, not a problem for this review.